### PR TITLE
Add documentation about the log's format

### PR DESCRIPTION
**Pull request by ecamacho**
I added a document about the format used in the xcactivitylog and a link to my talk when I explain the same topic to address the issue: https://github.com/spotify/XCLogParser/issues/10

### CODE REVIEW

File: docs/Xcactivitylog Format.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# The Xcactivitylog format
+
+Xcode logs are stored in files with the extension `.xcactivitylog`. Those files are gzipped to save storage. The files are encoded using a format called `SLF`.
+
+## The SLF format
+
+This information was first documented by Vincent Isambart in this [blog post](https://techlife.cookpad.com/entry/2017/12/08/124532).
+
+An `SLF` document starts with the header `SLF0`. After the header, the document has a collection of encoded values. The SLF encoding format supports these types:
```
Comment (BalestraPatrick) [inline docs/Xcactivitylog Format.md]: "A `SLF`"

File: docs/Xcactivitylog Format.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+- No left, nor right hand side value
+
+### String
+
+- Character type delimiter: `"`
+- Example: `5"Hello`
+- Left hand side value: An `Integer` with the number of characters that are part of the `String`.
+- Right hand side value: The characters that are part of the `String`
+
+The number of characters count works as in `NSString` rather than in `String`: it counts the 16-bit code units within the string’s UTF-16 representation and not the number of Unicode extended grapheme clusters within the string like in Swift's `String`.
```
Comment (BalestraPatrick) [inline docs/Xcactivitylog Format.md]: maybe "count" should be omitted?

File: docs/Xcactivitylog Format.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+[type: "double", value: 580158292.767495],
+[type: "double", value: 580158295.086277],
+[type: "array", count: 12],
+[type: "classInstance", className: "IDEActivityLogSection"],
+[type: "string", value: "Xcode.IDEActivityLogDomainType.XCBuild.Preparation"],
+[type: "string", value: "Prepare build"],
+[type: "string", value: "Prepare build"],
+```
+
+The first integer  is the version of the `SLF` format used. In Xcode 10.x and 11 Beta, the version is 10. The values after the version are the actual content of the log.
```
Comment (BalestraPatrick) [inline docs/Xcactivitylog Format.md]: double space between "integer" and "is"

**Comment by ecamacho** [comment]
Thanks! @BalestraPatrick fixed.

**Pull request merged**